cms/src/main/java/org/onehippo/taxonomy/plugin/ParameterisedTaxonomyBrowser.java

```diff
@@ -0,0 +1,137 @@
+package org.onehippo.taxonomy.plugin;
```

Comment:
The original TaxonomyBrowser, which this depends upon, has package level fields that we need to reference, hence it's been implemented in the same package. I would prefer to move everything into our own package, but it would require effectively copying the whole code file, rather than just extending where necessary. I am happy to talk about this as it has occurred in a few different places.

Reply:
Sure, let's talk about this, thanks